src/pages/DelegatoryValidator/StakingBar.tsx

```diff
@@ -12,12 +12,17 @@ import ContentBox from "../../components/IndividualPageContent/ContentBox";
 import {getFormattedBalanceStr} from "../../components/IndividualPageContent/ContentValue/CurrencyValue";
 import StakeDialog from "./StakeDialog";
 import ArrowCircleUpIcon from "@mui/icons-material/ArrowCircleUp";
+import {Types} from "aptos";
 
 type ValidatorStakingBarProps = {
   validator: MainnetValidatorData;
+  accountResource?: Types.MoveResource | undefined;
```

Comment:
decide to pass in `accountResource` even though right now we only need unlock time, in the future when more data sources coming in, we might need `accountResource` in other use cases.